server/models/ClubProfileModel.js

```diff
@@ -0,0 +1,41 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const clubProfileSchema = new Schema({
+    name: { type: String, required: true },
+    description: { type: String, required: true },
+    lastUpdated: { type: Date, default: Date.now() },
+    imageUrl: { type: String, default: '' },
+    memberRange: { type: String, required: true },
+    acceptingMembers: { type: Boolean, required: true },
+    applicationRequired: { type: Boolean, required: true },
+    meetingFrequency: { type: String, required: true },
+    socialLinks: {
+        type: Map,
+        of: String,
+        default: {
+            facebook: '',
+            email: '',
+            website: '',
+        }
```

Comment:
on the mockup of the club registration form, the possible fields for social links are "Website", "Facebook", "Instagram", “Link to join mailing list", and potentially email; let's stick to that for now!